misc/openapi-ballerina/modules/ballerina-client-generator/src/main/java/org/ballerinalang/client/generator/CodeGenerator.java

```diff
@@ -40,31 +40,30 @@
 
     /**
      * Generates code(client, server etc) for ballerina source provided  in <code>definitionPath</code>.
-     * <p>Method can be used for generating OAS 3.0.0, Swagger or Ballerina Client</p>
+     * <p>Method can be used for generating OAS 3.0.0 or Ballerina Client</p>
      *
      * @param type    Output type. Following types are supported
      *                <ul>
      *                <li>client</li>
      *                <li>openapi</li>
-     *                <li>swagger</li>
      *                </ul>
      * @param context Context details for generating the client
      * @return generated source is string representation of generated source.
-     * @throws CodeGeneratorException when file operations fail
+     * @throws ClientGeneratorException when file operations fail
      */
     public static String generateOutput(GeneratorConstants.GenType type, ClientContextHolder context)
-            throws CodeGeneratorException {
+            throws ClientGeneratorException {
         String output = "";
         switch (type) {
             case CLIENT:
                 return getConvertedString(context, GeneratorConstants.DEFAULT_CLIENT_DIR,
                         GeneratorConstants.CLIENT_TEMPLATE_NAME);
-            case OPENAPI:
+            case OPENAPI3:
                 return getConvertedString(context, GeneratorConstants.DEFAULT_OPEN_API_DIR,
                         GeneratorConstants.OPEN_API_TEMPLATE_NAME);
-            case SWAGGER:
-                return getConvertedString(context, GeneratorConstants.DEFAULT_SWAGGER_DIR,
-                        GeneratorConstants.SWAGGER_TEMPLATE_NAME);
+            case OPENAPI:
+                return getConvertedString(context, GeneratorConstants.DEFAULT_OPENAPI_DIR,
+                        GeneratorConstants.OPENAPI_TEMPLATE_NAME);
```

Comment:
OPENAPI_TEMPLATE_NAME and OPEN_API_TEMPLATE_NAME should be named after the version of the spec